ui/components/requirements/requirement.jsx

```diff
@@ -15,13 +16,37 @@ export default function Requirement({ requirement }) {
       <div className="req-text col col-10">
         { reqTexts }
         <div className="clearfix mt3">
-          <div className="applies-to mr2">
+          <div className="applies-to mr2 metadata">
             Applies to: [not implemented]
           </div>
-          <div className="sunset-date">
-            Sunset date by { requirement.policy.sunset || 'none' }
+          { requirement.issuing_body ? (
+            <div className="issuing-body metadata">
+              Issuing body: { requirement.issuing_body }
+            </div>
+          ) : (
+            ''
+          ) }
```

Comment:
Almost! JSX is creating objects behind the scenes, so we can't do straight string interpolation, but we could do:
```jsx
function Metadata({ className, key, value }) {
    if (!value) {
        return null;
    }
    return (
        <div className={`${className} metadata`} >
            {`${key} : ${value}`}
        </div>);
}
....
<someContainer>
    <Metadata className="issuance" key="Policy issuance" value={requirement.policy.omby_policy_id} />
</someContainer>
```